src/pages/Donors/Donors.tsx

```diff
@@ -1,16 +1,63 @@
 import {
   IonButton,
   IonButtons,
+  IonContent,
   IonHeader,
+  IonItem,
+  IonLabel,
+  IonList,
   IonPage,
   IonTitle,
   IonToolbar,
 } from "@ionic/react";
+import {
+  Maybe,
+  Message,
+  useCustomerConversationsListQuery,
+} from "generated/graphql";
 import useAuth from "hooks/useAuth/useAuth";
 import React from "react";
 
+interface LatestMessagePreviewProps {
+  messages: Array<Maybe<{ __typename?: "Message" } & Pick<Message, "content">>>;
+}
+const LatestMessagePreview: React.FC<LatestMessagePreviewProps> = ({
+  messages,
+}) => {
+  const latestMessage = messages[0];
+
+  if (latestMessage) {
+    const contentType = latestMessage.content.type;
+    if (contentType === "text") {
+      return <p>{latestMessage.content.text}</p>;
+    }
+
+    return <p>{`Sent ${contentType}`}</p>;
+  }
+  return null;
+};
+
+interface ConversationItemProps {
+  displayName: string;
+  messages: Array<Maybe<{ __typename?: "Message" } & Pick<Message, "content">>>;
+}
+const ConversationItem: React.FC<ConversationItemProps> = ({
+  displayName,
+  messages,
+}) => {
+  return (
+    <IonItem>
+      <IonLabel>
+        <h2>{displayName}</h2>
+        <LatestMessagePreview messages={messages} />
+      </IonLabel>
+    </IonItem>
+  );
+};
```

Comment:
Can probably extract that into its own file + tests

Reply:
Also you can maybe pass the whole conversation to the component, it might grow to require more things that `displayName` and `messages`

e.g

`<ConversationItem key={conversation.id} conversation={conversation} />`

To ensure props are present on the record you can probably use the `Pick<>` typescript type

e.g

`type ConversationItemProps = Pick<Conversation, 'customer' | 'messages'>`